app/views/pages/api_client_timeout.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l"><%= t("pages.api_client_timeout.title") %></h1>
+    <p class="govuk-body">
+      <%= t("pages.api_client_timeout.body_text") %>
+    </p>
+
+    <p class="govuk-body">
+      <%= t("pages.api_client_timeout.email_help",
+            team_name: t("service_name"),
+            email: support_email(Rails.application.config.support_email)).html_safe %>
```

Comment:
I've seen this somewhere else. I think it might be nice to improve the `support_email` helper with `support_email(name: nil)` for when we don't want to pass a name, and document it in the helper specs.